scripts/test.sh

```diff
@@ -180,4 +181,10 @@ if [[ $TEST_WITH_TESTSUITE = 1 ]]; then
     fi
 fi
 
+#Make docs
+if [[ $TEST_WITH_DOCS = 1 ]]; then
+   echo ">> Running make docs"
+   make docs
```

Comment:
The `if [[ $TEST_DRYRUN != 1 ]]; then` check is missing

Reply:
For consistency with the other cases, it would be good to use `run_make`, `add_failure`, and setting `+x`/`-x`.

Reply:
Sure.That makes sense.

Reply:
SORRY...so many unintended commits...still on my way to getting along with git. I'll practice somewhere else first next time, and I'll clean this mess up later.

Reply:
Sounds fine, let us know if you need help!

Reply:
ok, I'll try to figure it out by myself first. If it takes too much time unnecessarily, I'll just stop and turn to you for help. Thank you for your patience!